app/views/overview/_list-row-actions.html

```diff
@@ -15,7 +15,17 @@
               <a href="" ng-click="row.startBuild(row.buildConfigs[0])">Start Build</a>
             </li>
             <li ng-if="'deploymentconfigs/instantiate' | canI : 'create'" role="menuitem">
-              <a href="" ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href=""
+                ng-if="row.canDeploy()"
+                ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href=""
+                ng-if="!(row.canDeploy())"
+                class="disabled-link"
+                disabled>
```

Comment:
Cool, i'll swap. I did a little research briefly and found some things implying that redundant aria can be dropped, but ill roll with consistent.